pkg/model/dev.go

```diff
@@ -635,6 +639,17 @@ func (dev *Dev) setTimeout() error {
 	return nil
 }
 
+func (dev *Dev) validateSync() error {
+	for _, folder := range dev.Sync.Folders {
+		validPath, err := os.Stat(folder.LocalPath)
+		if !validPath.IsDir() && err != nil {
+			return err
```

Comment:
If err == nil and !validatePath.isDir() there is no error but it should fail because if it's a file its forbidden.
I would return a user error saying that you can only use folders in sync and a hint to https://okteto.com/docs/reference/manifest/#sync-string-required

Reply:
So this instead:

```go
if !valudPath.IsDir() && err == nil {}
```